release_notes.md

```diff
@@ -5,3 +5,38 @@
 The file describes all the changes, that are affected the usage of the application.
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
+
+## [0.1.0] - [29.05.2023]
+
+### New features
+
+`Quicklook`: Users can check a list of pet sitters available for pet owners. Each list row includes an avatar, full name, price, short bio, and pet sitter rating.
```

Comment:
```suggestion
`Quicklook`: Pet owners can see a list of available pet sitters. <IS THE LIST STATIC/DYNAMIC?> For every pet sitter the following information is available: an avatar icon, full name, current rate, short bio, and rating.
```

Reply:
You can scroll the list up and down. So it's dynamic list.
Thank you for suggestion. Fixed it.

Reply:
Thanks for the explanation! In this release it is fixed list.